modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,30 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section explains the similarities and differences in these Mule versions.
+
+Policies in Mule 4
+Classloader isolation exists between the application, the runtime and connectors, and policies. To fulfill the isolation requirement, policies are self-contained and packaged to include libraries needed for execution, similar to an application. Additionally, certain changes to the packaging, size, and scope of a policy are introdcued in Mule 4.
+
+[%header,cols="40a,50a,50a"]
+|===
+| Component | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
+| Policies distribution | Policies cannot be distributed outside of runtime | You can distribute policies outside of the runtime, which simplifies upgrades 
+| Policies  |           | All policies are non-blocking (what does this mean?)  
+| Policy execution | All polciies can be reordered and executed first | All policies, except CORS, can be reordered 
+| Resource Level Policy Support | Restricted to RAML-based APIs | Support extended to HTTP-based API, along with RAML-based APIs 
+| What is the component here?  |  | classloader isolation exists between application, runtime, connectors and policies 
```

Comment:
This line is incomplete: 

Component: Class-loading isolation. For Mule 3, no class-loading isolation is supported. For Mule 4: what is currently there.

Benefit: see this doc: https://docs.mulesoft.com/mule-runtime/4.2/about-classloading-isolation

Reply:
Done! Thanks!